Python/setup.py

```diff
@@ -39,31 +39,50 @@
 '''
 
 import os
+from pathlib import Path
+import re
```

Comment:
Neither are required. Use of pathlib might be nicer in many instances but this may not be one of them.

Reply:
Removed new imports.